migration/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Probably best to keep consistency with `npm` throughout the repo?

Reply:
oh, yeah. I like yarn, that's my default. will undo.